content/docs/04-clusters/05-imported-clusters/10-cluster-import.md

```diff
@@ -0,0 +1,245 @@
+
+
+4. Fill out the required information and make the proper selections:
+   * Cluster Name - The name of the cluster you want to import.
+   * Cloud Type - Select the infrastructure environment your cluster resides in. Select **Generic** if the environment list doesn't contain your specific environment but be aware of the limitations with generic clusters.
+   * Proxy - Optional and only available for generic clusters. Specify a network proxy address or DNS value.
+   * No Proxy - Optional and only available for generic clusters. Specify a no proxy address or DNS value.
+
+5. Select **Read-only mode** and click on **Create & Open Cluster Instance** to start the import action.
+
+
+6. You will be redirected to the cluster details page. A set of instructions with commands is displayed on the right-hand side of the screen. You will need to issue the following commands to complete the import process. 
+
+   <br />
+
+
+   ![A view of the cluster details page with the sidebar instructions box](/clusters_imported-clusters_read-only-instructions.png)
+
+   <br />
+
+7. We recommend you install the metrics server so that Palette can expose and provide you with information about the cluster. Installing the metrics server is not required but is needed for Palette to expose cluster metrics. Open a terminal session and issue the commands below against the Kubernetes cluster you want to import if you want to enable the metrics server.
+
+   <br />
+
+   ```shell
+   helm repo add bitnami https://charts.bitnami.com/bitnami && \
+   helm install my-release bitnami/metrics-server
+
+
+8. To install the Palette agent, issue the command displayed in the cluster details page  **Install the** read-only agent** section against the Kubernetes cluster you want to import. The command is customized for your cluster as it contains the assigned cluster ID. Below is an example output of the install command.
+
+   <br />
+
+   ```shell hideClipboard
+   kubectl apply -f https://api.spectrocloud.com/v1/spectroclusters/6491d4a94c39ad82d3cc30ae/import/manifest
+   ```
+
+   Output
+   ```shell hideClipboard
+   namespace/cluster-6491d4a94c39ad82d3cc30ae created
+   serviceaccount/cluster-management-agent created
+   clusterrole.rbac.authorization.k8s.io/read-only-mode created
+   clusterrolebinding.rbac.authorization.k8s.io/read-only-mode created
+   configmap/log-parser-config created
+   configmap/upgrade-info-8kfc2m8mt8 created
+   configmap/version-info-kbk5hk992f created
+   secret/spectro-image-pull-secret created
+   priorityclass.scheduling.k8s.io/spectro-cluster-critical created
+   deployment.apps/cluster-management-agent-lite created
+   configmap/cluster-info created
+   configmap/hubble-info created
+   secret/hubble-secrets created
+   ```
+
+9. Once the Palette agent completes the initialization, the side view drawer on the right will disappear, and your cluster will transition to a status of **Running** after a few moments.
```

Comment:
When the Palette agent completes initializing, the cluster import procedures at right will disappear, and your cluster will transition to **Running** status within a few minutes.